rainbowstream/colorset/config

```diff
@@ -24,7 +24,7 @@
     // 'conversation': max tweet in a thread
     "CONVERSATION_MAX" : 30,
     // 'quote' format
-    "QUOTE_FORMAT" : "#comment RT #owner: #tweet",
+    "QUOTE_FORMAT" : "#comment https:\/\/twitter.com\/#owner\/status\/#tid",
```

Comment:
One thing to note: even if we change value here it will not affect already installed user. I preserved their customized config file every update time. This can only help new users.